modules/new_user/cog.py

```diff
@@ -1,8 +1,7 @@
-
 import os
 from discord.utils import get
 from discord.ext import commands
-from modules.new_user.new_user import is_unassigned, add_role, remove_unassigned, change_nick, get_id
+from modules.new_user.new_user import is_unassigned, add_role, switch_unassigned, change_nick, get_id
 
 class new_user(commands.Cog):
```

Comment:
Please follow naming conventions outlined in the style guide. I.e. it should be NewUserCog